pkg/api/postage.go

```diff
@@ -300,10 +301,15 @@ func (s *Service) postageGetStampHandler(w http.ResponseWriter, r *http.Request)
 		jsonhttp.BadRequest(w, "invalid batchID")
 		return
 	}
-
-	issuer, err := s.post.GetStampIssuer(id)
-	if err != nil {
-		s.logger.Debug("get stamp issuer: get issuer failed", "batch_id", hex.EncodeToString(id), "error", err)
+	var issuer *postage.StampIssuer
+	for _, stampIssuer := range s.post.StampIssuers() {
+		if bytes.Equal(id, stampIssuer.ID()) {
+			issuer = stampIssuer
+			break
+		}
+	}
+	if issuer == nil {
+		s.logger.Debug("get stamp issuer: get issuer failed", "batch_id", hex.EncodeToString(id), "error")
```

Comment:
There is no error here. Remove the error key.

Also, create a logger here with the API name:
`logger := s.logger.WithName("get_stamp_by_id").Build()`

Then you can remove the `"get stamp issuer:"` from the log message. Check the simplify API PR from peter for examples.

Reply:
Done